diff -urp libcap-ng-0.7.3/utils/pscap.c libcap-ng-0.7.4/utils/pscap.c
--- libcap-ng-0.7.3/utils/pscap.c	2012-11-09 14:45:43.000000000 -0500
+++ libcap-ng-0.7.4/utils/pscap.c	2012-11-30 10:06:42.000000000 -0500
@@ -142,8 +142,6 @@ int main(int argc, char *argv[])
 				fclose(f);
 			}
 			
-			len = read(fd, buf, sizeof buf - 1);
-			close(fd);
 			if (header == 0) {
 				printf("%-5s %-5s %-10s  %-16s  %s\n",
 				    "ppid", "pid", "name", "command",
