Index: /trunk/src/test/thread_test.c
===================================================================
--- /trunk/src/test/thread_test.c	(revision 80)
+++ /trunk/src/test/thread_test.c	(revision 81)
@@ -46,4 +46,10 @@
 int main(void)
 {
+	// This test must be run as root which naturally has all capabilities
+	// set. So, we need to clear the capabilities so that we can see if
+	// the test works.
+	capng_clear(CAPNG_SELECT_CAPS);
+	capng_apply(CAPNG_SELECT_CAPS);
+
 	printf("Testing thread separation of capabilities\n");
 	pthread_create(&thread1, NULL, thread1_main, NULL);
